build.gradle.kts

```diff
@@ -58,3 +63,59 @@ jsonSchema2Pojo {
 	// What type to use instead of string when adding string properties of format "date-time" to Java types
 //	dateTimeType = "java.time.LocalDateTime"
 }
+
+val jacocoExclude = listOf("**/generated/**",
+		                   "**/api/**",
+		                   "**/cma/**")
+val percentageCoverage = ".30".toBigDecimal()
```

Comment:
How about calling it `minimumCoverage` or `coverageThreshold` to differentiate it from the coverage figure?

Reply:
Using minimumCoverage good call.

Reply:
👍